Ex1.java

```diff
@@ -1,3 +1,6 @@
+//Please review my code!
+//Ex1
+
 import java.util.HashMap;
 
 public class Ex1 {
```

Comment:
Because it looks like you merged some code in before you sent the pull request, I can't comment on the right lines, so I'll do it all in one comment.

This code looks pretty good. However, you do an s.toLowerCase() inside your loop. s.toLowerCase() operates on all characters of the string, taking time O(n) where n is the length of the string. You then also have that in a loop which does n iterations, making your whole algorithm O(n^2). Can you make a change to make it O(n)? If this doesn't make sense just ask me to clarify.